host-info.ts

```diff
@@ -24,3 +26,43 @@ export function isLinux() {
 export function isLinux64(): boolean {
 	return isLinux() && process.config.variables.host_arch === "x64";
 }
+
+export function dotNetVersion(message: string) : IFuture<string> {
+	var result = new Future<string>();
+	var Winreg = require("winreg");
+	var regKey = new Winreg({
+		hive: Winreg.HKLM,
+		key:  '\\Software\\Microsoft\\NET Framework Setup\\NDP\\v4\\Client'
+	});
+	regKey.get("Version", (err: Error, value: any) => {
+		if (err) {
+			result.throw(new Error(message));
+		} else {
+			result.return(value.value);
+		}
+	});
+	return result;
+}
+
+export function isDotNet40Installed(message?: string) : IFuture<boolean> {
+	return (() => {
+		try {
+			dotNetVersion(message || "An error occurred while reading the registry.").wait();
+			return true;
+		} catch (e) {
+			return false;
+		}
+	}).future<boolean>()();
+}
+
+export var hostCapabilities: { [key: string]: IHostCapabilities } = {
```

Comment:
this var is already part of appbuilder-cli and it is not used by nativescript-cli. Is there a reason to add it here? I couldn't find any place where you've used it in nativescript-cli